### PR TITLE
fix refresh_token issue

### PR DESCRIPTION
**Pull request by DzianisMakeichyk**
Fix issue #121

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -8,15 +8,43 @@ const api = got.extend({
...
+  return api
+    .post("auth/v3/authByEmail", {
+      json: {
+        device_type: "ANDROID",
+        email: credentials.email,
+      },
+    });
+  }
+
+function authPoll(polling_id) {
```
Comment (beatbrot) [inline lib/api.js:31]: It seems like this method is not called anywhere. Surely this PR can't fix our problem, right?

**Review by beatbrot — CHANGES_REQUESTED** [review]
I don't really understand this PR. It doesn't seem to fix the actual login process.

This PR might only work if you already have a valid refresh token from the past.

**Comment by c0lumb0** [comment]
> I don't really understand this PR. It doesn't seem to fix the actual login process.
> 
> This PR might only work if you already have a valid refresh token from the past.

Yup, just gave it a try. Works when there is a valid refresh token from the past.

**Comment by DzianisMakeichyk** [comment]
@c0lumb0 @c0lumb0 Hard to say anything, cause this code is copy-pasted from the issue and this code helped for me :)

**Comment by c0lumb0** [comment]
> @c0lumb0 @c0lumb0 Hard to say anything, cause this code is copy-pasted from the issue and this code helped for me :)

For me it worked when I added the tokens again...I guess you also still have them in your config? :)

**Comment by marklagendijk** [comment]
I merged #123 instead.

**Pull request closed without merging**